modules/telemetry-what-information-is-collected.adoc

```diff
@@ -29,6 +29,7 @@ endif::virt-cluster[]
 * Events for all namespaces listed as "related objects" for a degraded Operator
 * Configuration details that help Red Hat Support to provide beneficial support for customers. This includes node configuration at the cloud infrastructure level, host names, IP addresses, Kubernetes pod names, namespaces, and services.
 * Information about the validity of certificates
+* Number of builds by build strategy type
```

Comment:
The added line looks good to me, but the bullet point two points before ends with a full-stop while all other points don't. I understand the logic but it doesn't look to beautiful. I would consider forming it to another "including xy" sentence for consistency.

Reply:
Oh just found that for consistency reasons, maybe it should be "The number"

Reply:
Updated. Thanks.